canonical-websites/www.ubuntu.com
# remove old homepage takeover/takeunder partials/css

## Done

* removed all but current and generic takeovers and takeunders
* removed related css

## QA

1. go to the homepage
2. see that is looks fine
3. go to https://github.com/ubuntudesign/ubuntu.com-takeover-archive and see that it looks complete

## Issue / Card

Fixes #1510
